ci/azure/on_master_success

```diff
@@ -5,7 +5,7 @@ set +x
 
 set -e
 
-sudo apt-get update -y
+sudo apt-get update -y --allow-releaseinfo-chang
```

Comment:
Shouldn’t this be `—allow-releaseinfo-change`?

Reply:
Yep! fixed.